docs/DeveloperGuide.md

```diff
@@ -1432,3 +1432,58 @@ testers are expected to do more *exploratory* testing.
 
    1. Test case: `data/jobfestgo.json` was corrupted.
       Expected: JobFestGo starts with an empty event, contact, and task lists. All events, contacts and tasks are cleared.
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
+
+Given below are the planned enhancements for the app for future releases.
+
+1. **Make error messages more useful**
+
+   Currently, the error messages for certain commands are not very useful.
+   For example, if the user enters an invalid date such as `2024-04-31` `add_task`,
+   the error message is `Date should be valid and in format YYYY-MM-DD not before today's date`.
+   This is not very useful as the user does not know why the date is wrong.
+   The error message should be more specific, such as `Entered date is invalid. April 2024 should have DD between 1 and 30 inclusive`.
+
+2. **Allow emails to have top-level domains (TLDs) other than `.com`**
+
+   The email field only allows `.com` TLD in the current implementation.
+   This should be changed to allow other TLDs such as `.org`, `.edu`, `.net`, etc.
+
+3. **Allow special characters to be entered in `EventName` and `ContactName`**
+
+    Currently, special characters other than spaces are not allowed in `EventName` and `ContactName`.
+    This should be changed to allow these characters to allow more valid names to be entered.
```

Comment:
The phrasing here can be improved: `This should be changed to accept these characters to allow for a broader range of valid names.`

Reply:
Yup agreed! Thank you for the suggestion.